classes/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :first_name, :last_name, :items
+  attr_reader :id
+
+  def initialize(first_name, last_name)
+    @first_name = first_name
+    @last_name = last_name
+    @id = Random.rand(1..1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
```

Comment:
[Optional Suggestion] :
Prevents repeated items.
```suggestion
    @items.push(item) unless @items.include(item)
```